components/emoji_picker/components/emoji_picker_item.jsx

```diff
@@ -49,59 +45,61 @@ export default class EmojiPickerItem extends React.Component {
         }
     }
 
-    handleMouseOver() {
+    handleMouseOver = () => {
         if (!this.props.isSelected) {
             this.props.onItemOver(this.props.categoryIndex, this.props.emojiIndex);
         }
-    }
+    };
+
+    handleMouseOverThrottle = debounce(this.handleMouseOver, EMOJI_LAZY_LOAD_SCROLL_THROTTLE, {leading: true, trailing: true});
 
-    handleClick() {
+    handleClick = () => {
         this.props.onItemClick(this.props.emoji);
-    }
+    };
 
     render() {
         const {emoji} = this.props;
-        let item = null;
+
         let itemClassName = 'emoji-picker__item';
-        itemClassName += this.props.isSelected ? ' selected' : '';
+        if (this.props.isSelected) {
+            itemClassName += ' selected';
+        }
+
         let spriteClassName = 'emojisprite';
         spriteClassName += ' emoji-category-' + emoji.category + '-' + emoji.batch;
         spriteClassName += ' emoji-' + emoji.filename;
 
+        let image;
+        let handleMouseOver;
         if (emoji.category && emoji.batch) {
-            item = (
-                <div
-                    className={itemClassName}
-                    ref={this.emojiItemRef}
-                >
-                    <div>
-                        <img
-                            src='/static/images/img_trans.gif'
-                            className={spriteClassName}
-                            onMouseOver={this.handleMouseOverThrottle}
-                            onClick={this.handleClick}
-                        />
-                    </div>
-                </div>
+            image = (
+                <img
+                    src='/static/images/img_trans.gif'
+                    className={spriteClassName}
+                />
             );
+            handleMouseOver = this.handleMouseOverThrottle;
         } else {
-            item = (
-                <div
-                    className={itemClassName}
-                    ref={this.emojiItemRef}
-                >
-                    <div>
-                        <img
-                            src={EmojiStore.getEmojiImageUrl(emoji)}
-                            className={spriteClassName}
-                            onMouseOver={this.handleMouseOver}
-                            onClick={this.handleClick}
-                        />
-                    </div>
-                </div>
+            image = (
+                <img
+                    src={EmojiStore.getEmojiImageUrl(emoji)}
+                    className={spriteClassName}
```

Comment:
`spriteClassName` is a bit misleading to me since it's not really from a sprite sheet image. Maybe just set a class name, like:
```
className={'emoji-category-custom'}
```
and then with that, we could eliminate `-undefined` at `sass/components/_emoticons.scss`